src/System.Private.CoreLib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -423,15 +423,23 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Reflection\CustomAttributeFormatException.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Reflection\DefaultMemberAttribute.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Reflection\Emit\AssemblyBuilderAccess.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Reflection\Emit\EventToken.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Reflection\Emit\FieldToken.cs" />
```

Comment:
As far as I can tell these don't exist in CoreRT now. Would it make sense to guard this with some new condition (eg. `FeatureReflectionEmit`)?

Reply:
If we added it to .NET Standard, CoreRT will be happy to pick this up. I don't see it here, but maybe it's not up to date: https://apisof.net/catalog/System.Reflection.Emit.FieldToken

Reply:
Looks like it is in .NET Standard 2.1 - https://github.com/dotnet/standard/blob/master/src/netstandard/ref/System.Reflection.Emit.cs